include/groonga/expr.h

```diff
@@ -96,6 +96,38 @@ grn_expr_append_const_float(
 GRN_API grn_rc
 grn_expr_append_op(grn_ctx *ctx, grn_obj *expr, grn_operator op, int nargs);
 
+/**
+ * \brief Extract keywords from an expression.
+ *
+ * For example usage:
+ * ```c
+ *  grn_obj keywords;
+ *  GRN_PTR_INIT(&keywords, GRN_OBJ_VECTOR, GRN_ID_NIL);
+ *  // You don't need to unlink keyword because it is owned by \p expr.
```

Comment:
Can we use `\p` in comment in pre-formatted part?

Reply:
fix: [b427faa](https://github.com/groonga/groonga/pull/2264/commits/b427faa25c940cb46cf0efd21608e859905cf5ec) Sorry, we couldn't. it doesn't work as follows.

![Screenshot from 2025-04-02 13-02-01](https://github.com/user-attachments/assets/62d9f1f5-dcd2-4398-8dda-7d266f27daa6)